### DISCUSSION TITLE
ImportError: cannot import name 'register_all_modules'

### DISCUSSION THREAD

**Discussion by w1125** [open-mmlab/mmdeploy#1820] [General]
Traceback (most recent call last):
  File "tools/train.py", line 12, in <module>
    from mmocr.utils import register_all_modules
ImportError: cannot import name 'register_all_modules'

![image](https://user-images.githubusercontent.com/101447162/222033675-f5857ea4-e753-4acc-aa41-8bded970d804.png)
模块注册都正常，不知道为什么显示无法导入

**Comment by lvhan028** [comment]
mmdeploy has nothing to do with training a model. 
I suggest you open this discussion in [mmocr](https://github.com/open-mmlab/mmocr) repo
